app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -1,28 +1,30 @@
+<p>A teaching internship gives you the opportunity to spend 3 weeks in a school and find out what teaching is really like. You’ll experience being in a classroom, gain new skills, and be paid £300 per week.</p>
 
-<p>You may be eligible for a paid teaching internship if you're:</p>
+<p>Teaching internships take place during June and July this year. Places fill up quickly.</p>
+
+<h2>Subjects you can do your internship in</h2>
+
+<p>Teaching internships are only available for certain subjects. You can do an internship in:</p>
 
 <ul>
- <li>in any year of a science, technology, engineering or maths (STEM) undergraduate degree</li>
-  <li>interested in teaching secondary maths or physics</li>
+ <li>computing</li>
+  <li>maths</li>
+  <li>physics</li>
+  <li>languages (north-east schools only)</li>
```

Comment:
Should this be North East rather than north-east as that is how the region is labelled in the data?